app/views/RoomsListView/index.tsx

```diff
@@ -456,17 +479,33 @@ class RoomsListView extends React.Component<IRoomsListViewProps, IRoomsListViewS
 							: // @ts-ignore
 							  () => navigation.toggleDrawer()
 					}
-					badge={() => (notificationPresenceCap ? <HeaderButton.BadgeWarn /> : null)}
+					badge={() => getBadge()}
```

Comment:
```suggestion
					badge={getBadge}
```

Reply:
It's a choice, really.
It doesn't change anything and I've been thinking in the cases when the callback has a param, like `onPress` and its `nativeEvent`.
Of course we use it only if needed, but it's more like an explicit indicator of no props.
Again, just a choice 😂